Formula/php.rb

```diff
@@ -64,9 +64,6 @@ class Php < Formula
   patch :DATA
 
   def install
-    # Ensure that libxml2 will be detected correctly in older MacOS
-    ENV["SDKROOT"] = MacOS.sdk_path if MacOS.version == :el_capitan || MacOS.version == :sierra
-
```

Comment:
Is this fixed now?

Reply:
I don't have a El Capitan or Sierra environment to test this. My understanding is that those versions are not supported by Homebrew anymore. But I can re-add the line. It's an unrelated change, after all.

Reply:
@derrabus El Capitan and Sierra are supported on a best-effort basis. see https://docs.brew.sh/Installation#2

> 10.14 or higher is recommended. 10.9–10.13 are supported on a best-effort basis.

Reply:
Thanks @nabihnadhif. I have reverted this change.